src/features/views/components/ViewColumnDialog/ColumnGallery/index.tsx

```diff
@@ -126,28 +132,30 @@ const ColumnGallery: FunctionComponent<ColumnGalleryProps> = ({
           width="20%"
         >
           <List>
-            {categories.map((category, index) => (
-              <ListItem
-                key={index}
-                onClick={() => {
-                  setSearchString('');
-                  setIsSearching(false);
-                  if (choiceContainerRef.current) {
-                    const element = choiceContainerRef.current.querySelector(
-                      `#category-${index}`
-                    );
-                    element?.scrollIntoView({ behavior: 'smooth' });
-                  }
-                }}
-                sx={{ cursor: 'pointer', paddingY: 2 }}
-              >
-                <Typography>
-                  <Msg
-                    id={`misc.views.columnDialog.categories.${category.key}.title`}
-                  />
-                </Typography>
-              </ListItem>
-            ))}
+            {filteredCategories.map((category, index) => {
+              return (
```

Comment:
There is no need for this `return`. Just use a single-expression function instead (i.e. replace the `{` with a `(` on line 135).